tests/test_custom_checkers/test_pycodestyle_checker.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import pylint.testutils
+from astroid.astroid_manager import MANAGER
+
+from python_ta.checkers.pycodestyle_checker import PycodestyleChecker
+
+DIR_PATH = (
+    os.path.abspath(os.path.join(__file__, "../../.."))
+    + "/examples/custom_checkers/e9989_pycodestyle/"
+)
+
+
+class TestPycodestyleChecker(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = PycodestyleChecker
+    CONFIG = {"pycodestyle_ignore": ["E121", "E123", "E126", "E24", "E704", "W503", "W504"]}
```

Comment:
Since this will be combined with other tests for other pycodestyle errors, I think the most conservative thing to do is set `pycodestyle_ignore = []`. If for whatever reason an empty list doesn't work, pick just a single error to ignore.

Reply:
An empty list causes pycodestyle to use the DEFAULT_IGNORE value, so I just left E24 in the list.